setup.cfg

```diff
@@ -1,4 +0,0 @@
-[flake8]
-max-line-length = 120
```

Comment:
I think this removal causing your issues with the linter

Reply:
good catch